### ISSUE TITLE
experiment admin error

### ISSUE THREAD

**Issue by chhantyal** [mixcloud/django-experiments#141]
Sorry for creating another issue, but I am have problem using this app.

The admin gives following error (this example project included in the package which is running after #140):

`Data column(s) for axis #0 cannot be of type string×`

![see screenshot](https://cloud.githubusercontent.com/assets/862941/9467357/60cc9f5c-4b3b-11e5-88a8-4704ef3fc79f.png)

**Comment by sdcooke** [comment]
The charts work for us in production - I've definitely seen them not render in development before. Can you share the JSON you're seeing in the page when you get the error?

**Comment by matclayton** [comment]
Closing as we've not seen this happen and don't have repro steps.
